web-tester-fixture/pom.xml

```diff
@@ -7,7 +7,7 @@
   <version>12.0.2-SNAPSHOT</version>
   <packaging>iar-integration-test</packaging>
   <properties>
-    <project.build.plugin.version>12.0.1-SNAPSHOT</project.build.plugin.version>
```

Comment:
switched to use non-snapshot artifacts, don't think it will hurt us much.
maybe we're even closer to reality like this.